specs/wp-post-editor-spec.js

```diff
@@ -343,6 +344,69 @@ test.describe( `[${host}] Editor: Posts (${screenSize})`, function() {
 		} );
 	} );
 
+	test.describe( 'Schedule Basic Public Post @parallel @jetpack', function() {
+		this.bailSuite( true );
+		let publishDate;
+
+		test.it( 'Delete Cookies and Local Storage', function() {
+			driverManager.clearCookiesAndDeleteLocalStorage( driver );
+		} );
+
+		test.describe( 'Schedule a New Post', function() {
+			const blogPostTitle = dataHelper.randomPhrase();
+			const blogPostQuote = '“Worries shared are worries halved.”\n- Unknown';
+
+			test.it( 'Can log in', function() {
+				this.loginFlow = new LoginFlow( driver );
+				return this.loginFlow.loginAndStartNewPost();
+			} );
+
+			test.it( 'Can enter post title and content', function() {
+				this.editorPage = new EditorPage( driver );
+				this.editorPage.enterTitle( blogPostTitle );
+				this.editorPage.enterContent( blogPostQuote + '\n' );
+
+				return this.editorPage.errorDisplayed().then( ( errorShown ) => {
+					return assert.equal( errorShown, false, 'There is an error shown on the editor page!' );
+				} );
+			} );
+
+			test.it( 'Can schedule content for a future date (first day of second week next month)', function() {
+				let postEditorToolbarComponent = new PostEditorToolbarComponent( driver );
+				postEditorToolbarComponent.ensureSaved( { clickSave: true } );
+				let postEditorSidebarComponent = new PostEditorSidebarComponent( driver );
+				postEditorSidebarComponent.expandStatusSection();
+				postEditorSidebarComponent.chooseFutureDate();
+				postEditorSidebarComponent.getSelectedPublishDate().then( ( publishDateShown ) => {
+					publishDate = publishDateShown;
+				} );
+				postEditorSidebarComponent.closeStatusSection();
+				postEditorToolbarComponent = new PostEditorToolbarComponent( driver );
+				postEditorToolbarComponent.ensureSaved( { clickSave: true } );
+				return postEditorToolbarComponent.clickPublishPost();
+			} );
+
+			test.it( 'Can confirm scheduling post and see correct publish date', function() {
+				let editorConfirmationSidebarComponent = new EditorConfirmationSidebarComponent( this.driver );
```

Comment:
Should `this.driver` in this test block be `driver`? When I make that change this scheduled post test passes locally, without any trouble caused by a dirty editor state or other issues.

Reply:
Thanks, it makes sense for those not to contain `this`, but changing those the test still fails locally for me :(